README.md

```diff
@@ -149,12 +149,15 @@ The paths passed to the linters are absolute to avoid confusion in case they're
 }
 ```
 
-**NOTE:** Prior to `lint-staged@5`, when the `gitDir` option was present, file patterns used for filtering via minimatch had to be relative to the git root.
-But that's no longer necessary and the `gitDir` option is deprecated. Instead, `lint-staged` will now do the following:
+When mathcing, lint-staged will do the following
 
 * Resolve the git root automatically, no configuration needed.
 * Pick the staged files which are present inside the project directory.
 * Filter them using the specified glob patterns.
+* Resolve filtered paths relative to the git root and make them absolute.
+* Pass absolute paths to the linters as arguments
+
+**NOTE:** Lint-staged will pass _absolute_ paths to the linters commands to avoid any confusion in case they're executed in a different working directory (i.e. when your `.git` directory isn't the same as your `package.json` directory).
```

Comment:
I think we need to rework on this sentence:
>i.e. when your `.git` directory isn't the same as your `package.json` directory

although I understand what it means, but I think it's still confusing for new users

Reply:
What would you suggest?

Reply:
>(i.e. when your `.git` directory isn't in the same location to your `package.json`)

thoughts?

Reply:
Your suggestion does not seem to be different from what we have now. I'll stick with the current version.